test/views/templates/PaymentsHistoryChargeTemplateSpec.scala

```diff
@@ -171,6 +171,62 @@ class PaymentsHistoryChargeTemplateSpec extends ViewBaseSpec {
       }
     }
 
+    "there is a vat central assessment charge" should {
+
+      val model: PaymentsHistoryModel = PaymentsHistoryModel(
+        "VAT Central Assessment",
+        Some(LocalDate.parse("2018-01-12")),
+        Some(LocalDate.parse("2018-03-23")),
+        -123456,
+        Some(LocalDate.parse("2018-02-14"))
+      )
+
+      lazy val template = views.html.templates.paymentsHistoryCharge(model)
+      lazy implicit val document: Document = Jsoup.parse(
+        s"<table>${template.body}</table>"
+      )
+
+      "display the correct table row class" in {
+        element(Selectors.tableRow).attr("class") shouldBe ""
+      }
+
+      "display the correct charge title" in {
+        elementText(Selectors.chargeTitle) shouldBe messages("paymentsHistory.vatCentralAssessmentTitle")
```

Comment:
Could you write the message out here please instead of using the actual stored message. If there's a mistake in the message it won't get picked up in the unit tests